.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# See go/codeowners - automatically generated for confluentinc/apac-workshops:
+*	@javabrett
```

Comment:
@lyoung-confluent Seems like I (@javabrett ) and @ora0600 don't have access to this repo, so errors are highlighted here.  Does it matter?

Reply:
Strictly speaking no, GitHub's behavior is to just ignore invalid entries in `CODEOWNERS`. If you should have (write) access to the repo you could have one of the current owners add you and the error will go away.